Dockerfile

```
@@ -1,8 +1,15 @@
-FROM rust:latest
+FROM rust:latest as builder
 
 WORKDIR /usr/src/libreddit
 COPY . .
-
 RUN cargo install --path .
 
+
+FROM debian:buster-slim
+
+RUN apt-get update && apt-get install -y libcurl4 && rm -rf /var/lib/apt/lists/*
+COPY --from=builder /usr/local/cargo/bin/libreddit /usr/local/bin/libreddit
+RUN useradd --system --user-group --home-dir /nonexistent --no-create-home --shell /usr/sbin/nologin libreddit
+USER libreddit
+
```

Define Port https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#expose
```suggestion

EXPOSE 8080

```